GOVERNANCE.md

```diff
@@ -0,0 +1,91 @@
+# Aerie Project Governance
+
+The Aerie project aims to create an open source community that encourages transparency, contributions, and collaboration, but maintains sound technical and quality standards. Our goal is to build a community comprised of members across the the space community and beyond, including from private organizations, universities, Federaly Funded Research and Development Ceneters (FRDCs), other government organizations, and international organizations. 
+
+The project follows a fairly liberal contribution model where people and/or organizations who do the most work will have the most influence on project direction. Technical decision making will primarily be made through a "[consensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)" approach. 
+
+## Roles
+
+### User
+
+Anyone who has downloaded, deployed, or operated the Aerie system to meet a specific objective. Aeire was primarily designed for space mission activity planning, modeling, and simulation, but let us know if you've found other uses for it.  
+
+### Contributor
+
+Contributors include anyone that provides input to the project. This includes code, documentation, graphics, designs, or anything else that tangibly improves the project. We encourage you to start contributing right away by joining our [Discussions](https://github.com/NASA-AMMOS/aerie/discussions) or submitting an [Issue](https://github.com/NASA-AMMOS/aerie/issues). 
+ 
+### Collaborator
+
+Subset of contributors who have been given write access to one or more of the Aerie repositories. Both contributors and collaborators can propose changes to the project via pull requests, but only collaborators can formally review and approve (merge) these requests. Any contributor who has made a non-trivial contribution should be on-boarded as a collaborator in a timely manner. 
+
+If you are planning on making a substantial contribution to the project or feel as though you should be given write access to a repository, please reach out to TBD (some email list??)...
+
+## Technical Steering Committee
+
+A subset of the collaborators forms the Technical Steering Committee (TSC). The TSC has authority over the following aspects of this project:
+
+- Technical direction and guidance
+- Project governance and process 
+- Contribution policy
+- Conduct guidelines
+- Maintaining the list of collaborators
+
+The current list of TSC members is maintained TBD

```

Comment:
Flagging lines that have TBDs so we don't accidentally merge before they're filled in.

Reply:
I think this is okay to merge since we don't actually have these yet. I'll defer to @ewferg tho.

Reply:
Regarding that deferral, if you agree that it's okay to merge, @ewferg, then my feedback regarding the TBDs changes to a suggestion that the lines with 'TBD' be reworded to flow better, as the phrase `is maintained To Be Determined` was why I thought they were details that needed to be filled in before merging.

Reply:
I would like to go ahead and knock out the TBDs before we merge this in, hopefully in the next day or two.

Reply:
Went ahead and added a proposed TSC list.